docs/tutorials/run-kinto.rst

```diff
@@ -73,8 +73,8 @@ Then install the package using the default configuration:
 ::
 
     pip install kinto
-    wget https://raw.githubusercontent.com/Kinto/kinto/master/config/kinto.ini
-    pserve kinto.ini
+    wget https://raw.githubusercontent.com/Kinto/kinto/master/config/kinto.ini.sample -O kinto.ini
+    kinto --ini kinto.ini start
```

Comment:
I would add the sample in the egg anyway + @oak11 work on the init command to do something like:

    kinto init
    kinto start